savour-api-lib/src/message/sendMessage.ts

```diff
@@ -45,13 +53,20 @@ async function handleSendMessageRecord(request: SendMessageRequest): Promise<any
 		if (business) {
 			const businessNumber = business.messagingNumber;
       const subscribers = Object.entries(business.subscriberMap).filter(([_, subscriber]) => subscriber.subscribed);
-			const messagePromises = subscribers.map(([subscriberNumber, _]) => {
-				return twilio.sendMessage(businessNumber, subscriberNumber, campaign.message, shortLink)});
+			const messagePromises = subscribers.map(([subscriberNumber]) => twilio.sendMessage(businessNumber, subscriberNumber, campaign.message, shortLink));
```

Comment:
We should eventually move this out to its own accessor/library. For now it should be fine living in here though.